### ISSUE TITLE
ACC ServerManager File Structure / Pull Requests on different Repos

### ISSUE THREAD

**Issue by surtic86** [grimsi/accservermanager#17]
Hi there,

So i have don my first Pull Requests to your Repos and there i have a open Question. 

Right now the Server Backend Code and Frontend Code are in Separate Repos. On one way i like this and on another i don't. Because now i have to make two or three Seperate Pull Request (Backend, Frontend, API). And i think its get quite tricky also for you to manage, test them.

Would it be not better to have them all in one Repo just under Separates Folder?

backend/
frontend/
api/
README
usw.

**Comment by grimsi** [comment]
What you're referring to is a Monorepo if I understand correctly.I tried setting something like this up using maven multi-modules and I couldn't get it to work smoothly.I will take a look into it because you're correct that it will make testing and integrating much more convenient.

**Comment by surtic86** [comment]
Yes kindly also when i not would say it is a Monorepo because they are not really Different Projects as far as i Understand Monorepo (Wikipedia).

Just take your time. right now i have set it up and can make the Pull Request on every Repo.

**Comment by grimsi** [comment]
I wrote a short developer guide: https://github.com/grimsi/accservermanager/wiki/Development-Guide-Backend

If you want you can take a look at it and give feedback :)
